js/apps/admin-ui/src/components/table-toolbar/TableToolbar.tsx

```diff
@@ -35,7 +37,9 @@ export const TableToolbar = ({
   const onSearch = () => {
     if (searchValue !== "") {
       setSearchValue(searchValue);
-      inputGroupOnEnter?.(searchValue);
+      inputGroupOnEnter?.(
+        exactSearch ? searchValue : searchValue.toLowerCase(),
```

Comment:
I am not convinced this is the place to do this. Handling case-insensitivity is a concern of the search algorithm, not an implementation detail of how the value is set.

Reply:
oh I completely misunderstood you lol, I have fixed as you told me before